website/static/js/search.js

```diff
@@ -283,7 +283,8 @@ var ViewModel = function(params) {
         self.currentPage(1);
         self.category(alias);
         if (alias.name === 'SHARE') {
-            document.location = '/share/?' + $.param({q: self.query()});
+            var win = window.open(window.contextVars.shareUrl + 'discover?' + $.param({q: self.query()}), '_blank');
```

Comment:
Nice job remembering! Looks like it'll be fine as long as we are directing to a domain we control.

As noted [here](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/) just to be safe add in the following line before the `win.focus()`.

```js
win.opener = null;
```